charts/bookkeeper-operator/README.md

```diff
@@ -11,37 +11,41 @@ This chart bootstraps a [Bookkeeper Operator](https://github.com/pravega/bookkee
-> The name of the certificate (*webhookCert.certName*), the name of the secret created by this certificate (*webhookCert.secretName*), the tls.crt (*webhookCert.crt*) and tls.key (*webhookCert.key*) need to be specified against the corresponding fields in the values.yaml file, or can be provided with the install command as shown [here](#installing-the-chart).
-The values *tls.crt* and *tls.key* are contained in the secret which is created by the certificate and can be obtained using the following command
-```
-kubectl get secret <secret-name> -o yaml | grep tls.
-```
-
+  - An Issuer and a Certificate (either self-signed or CA signed) in the same namespace that the Bookkeeper Operator will be installed (refer to [this](https://github.com/pravega/bookkeeper-operator/blob/master/deploy/certificate.yaml) manifest to create a self-signed certificate in the default namespace)
 
 ## Installing the Chart
 
-To install the chart with the release name `my-release`:
+To install the bookkeeper-operator chart, use the following commands:
 
 ```
-$ helm install my-release bookkeeper-operator --set webhookCert.generate=false --set webhookCert.certName=<cert-name> --set webhookCert.secretName=<secret-name>
+$ helm repo add pravega https://charts.pravega.io
+$ helm repo update
+$ helm install [RELEASE_NAME] pravega/bookkeeper-operator --version=[VERSION] --set webhookCert.certName=[CERT_NAME] --set webhookCert.secretName=[SECRET_NAME]
 ```
+where:
+- **[RELEASE_NAME]** is the release name for the bookkeeper-operator chart
+- **[DEPLOYMENT_NAME]** is the name of the bookkeeper-operator deployment so created. (If [RELEASE_NAME] contains the string `bookkeeper-operator`, `[DEPLOYMENT_NAME] = [RELEASE_NAME]`, else `[DEPLOYMENT_NAME] = [RELEASE_NAME]-bookkeeper-operator`. The [DEPLOYMENT_NAME] can however be overridden by providing `--set fullnameOverride=[DEPLOYMENT_NAME]` along with the helm install command)
+- **[VERSION]** can be any stable release version for bookkeeper-operator from 0.1.3 onwards
+- **[CERT_NAME]** is the name of the certificate created as a prerequisite
+- **[SECRET_NAME]** is the name of the secret created by the above certificate
+
+This command deploys a bookkeeper-operator on the Kubernetes cluster in its default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
-The command deploys bookkeeper operator on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+>Note: If the bookkeeper-operator version is 0.1.2, webhookCert.certName and webhookCert.secretName should not be set. Also in this case, cert-manager and the certificate/issuer do not need to be deployed as prerequisites.
```

Comment:
here we can mention 0.1.2 or belw

Reply:
charts have been published only from version 0.1.2 onwards, which has been added in the readme